domain/src/main/java/life/qbic/domain/usermanagement/policies/PasswordPolicy.java

```diff
@@ -32,7 +34,7 @@ public static PasswordPolicy create() {
    */
   public PolicyCheckReport validate(char[] rawPassword) {
     if (rawPassword.length < MIN_LENGTH) {
-      return new PolicyCheckReport(PolicyStatus.FAILED, "Password shorter than 8 characters.");
+      return new PolicyCheckReport(PolicyStatus.FAILED, invalidPasswordMessage);
```

Comment:
same here as above. no need for a variable

Reply:
alright 👍